distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -726,6 +729,85 @@ type Table
         result = Table.new new_columns
         on_problems.attach_problems_after result problem_builder.to_vector
 
+    ## Replaces the first, last, or all occurrences of term with new_text in each text row of selected columns.
+       If `term` is empty, the function returns the table unchanged.
```

Comment:
```suggestion
    ## Replaces the first, last, or all occurrences of the `term` with 
       `new_text` in each text row of selected columns.
       
       If `term` is empty, the function returns the table unchanged.
```

Please try to keep the doc comments wrapped to 80 characters per line, at least where reasonably possible.

Also good to wrap variable names in `backticks`.